dev-client/src/screens/LoginScreen.tsx

```diff
@@ -55,7 +61,7 @@ export const LoginScreen = () => {
 
   useEffect(() => {
     if (loggedIn) {
-      navigation.replace('HOME', {});
+      navigation.replace('BOTTOM_TABS');
```

Comment:
question: do we ant the second parameter?

Reply:
nope! `BOTTOM_TABS` is a new screen in this PR which contains `HOME` as a subscreen

Reply:
although this does remind me there's one more regression i need to fix before merging this